### ISSUE TITLE
[ingestion/data-quality issue] Ingesting an SPDX SBOM with `LicenseRef-` license fails

### ISSUE THREAD

**Issue by mrizzi** [guacsec/guac#2165]
**Describe the bug**
In [PR-2164](https://github.com/guacsec/guac/pull/2164/files#diff-02d5624625d36a110ff0a0c481a7e0ce8089518ef73a45da419ab780afbd21f3R1021) the test has been changed to expect both `Inline` and `ListVersion` but this would cause the `Both Inline and ListVersion are provided` error during ingestion because of failure in validating license in: https://github.com/guacsec/guac/blob/f13bed16fc13607ced4269e5ff5b1ae13f48c9ec/pkg/assembler/graphql/resolvers/validation.go#L143 

**To Reproduce**
Basically the test's expectation is wrong so reverting it to not expect `ListVersion: ptrfrom.String("1.2.3")` would reproduce the issue.

**Expected behavior**
Not having a value for `ListVersion` field when `Inline` has a value

**GUAC version**
Current `main` branch, i.e. f13bed16fc13607ced4269e5ff5b1ae13f48c9ec

**Ingested document(s)**

`SPDX with custom licenses` test

**Additional context**

**Comment by pxp928** [comment]
ah good catch @mrizzi!
